### PR TITLE
fix(usm): add role img to shared link expiration tooltip icon

### PR DESCRIPTION
**Pull request by benjamin-shen**

### CODE REVIEW

File: src/features/unified-share-modal/SharedLinkSection.js [hunk 1]
```diff
@@ -462,31 +457,26 @@ class SharedLinkSection extends React.Component<Props, State> {
...
-                                        expiration: convertToMs(expirationTimestamp),
-                                    }}
-                                />
-                            }
+                            text={intl.formatMessage(messages.sharedLinkExpirationTooltip, {
+                                expiration: convertToMs(expirationTimestamp),
+                            })}
                         >
-                            <span className="shared-link-expiration-badge">
+                            <span className="shared-link-expiration-badge" role="img">
```
Comment (benjamin-shen) [inline src/features/unified-share-modal/SharedLinkSection.js:471]: Here's the accessibility change.

File: src/features/unified-share-modal/__tests__/__snapshots__/SharedLinkSection.test.js.snap [hunk 2]
```diff
@@ -28,30 +25,18 @@ exports[`features/unified-share-modal/SharedLinkSection should account for share
...
-                  defaultMessage="This link will expire and be inaccessible on {expiration, date, long}."
-                  id="boxui.unifiedShare.sharedLinkExpirationTooltip"
-                  values={
-                    Object {
-                      "expiration": 1519404618000000,
-                    }
-                  }
-                />
-              }
+              text="This link will expire and be inaccessible on {expiration, date, long}."
```
Comment (benjamin-shen) [inline src/features/unified-share-modal/__tests__/__snapshots__/SharedLinkSection.test.js.snap:34]: Should we change this default message to be more generic? (I don't think `{expiration, date, long}` will be replaced by a human-readable date, will it??)

**Review by QingyuChai — DISMISSED** [review]
lgtm!

File: src/features/unified-share-modal/__tests__/__snapshots__/SharedLinkSection.test.js.snap (hunk 2 above)
Comment (QingyuChai) [reply src/features/unified-share-modal/__tests__/__snapshots__/SharedLinkSection.test.js.snap:34]: that is a good point - I think this is worth addressing, but probably in a separate ticket

**Review by QingyuChai — APPROVED** [review]

**Review by tjuanitas — APPROVED** [review]

**Pull request merged**